docs/v0.7/en/greptimecloud/usage-&-billing/wcu-rcu.md

```diff
@@ -0,0 +1,89 @@
+A standard write capacity unit can write rows up to 1KB.
+For rows larger than 1KB, additional write capacity units are required.
+
+:::tip NOTE
+The capacity unit may be subject to change in the future.
+:::
+
+The following steps are used to determine the size of each request:
+
+1. Get the size of the data type of each column in the table schema. You can find details about the size of each data type in the [Data Types](/reference/sql/data-types.md) documentation.
+2. Sum up the sizes of all columns in the request. If a column is not present in the request, its size depends on the column's default value. If the default value is null, the size is 0; otherwise, it is the size of the data type.
+3. Multiply the sum by the number of rows to be written.
+
+Here's an example of how to calculate the WCU for a table with the following schema:
+
+```shell
++-------------+----------------------+------+------+---------------------+---------------+
+| Column      | Type                 | Key  | Null | Default             | Semantic Type |
++-------------+----------------------+------+------+---------------------+---------------+
+| host        | String               | PRI  | YES  |                     | TAG           |
+| idc         | String               | PRI  | YES  |                     | TAG           |
+| cpu_util    | Float64              |      | YES  |                     | FIELD         |
+| memory_util | Float64              |      | YES  |                     | FIELD         |
+| disk_util   | Float64              |      | YES  |                     | FIELD         |
+| ts          | TimestampMillisecond | PRI  | NO   | current_timestamp() | TIMESTAMP     |
++-------------+----------------------+------+------+---------------------+---------------+
+```
+
+You have a write request as following:
+
+```shell
+INSERT INTO system_metrics VALUES ("host1", "a", 11.8, 10.3, 10.3, 1667446797450);
+```
+
+Based on the size of the data types in your table schema, the size of each row is 38 bytes (5+1+8+8+8+8), and the WCU of this request is 1 according to the calculation algorithm.
+
+To reduce the WCU usage, use batched `INSERT` statements to insert multiple rows in a single statement, rather than sending a separate statement per row. For example:
+
+```shell
+INSERT INTO system_metrics
+VALUES
+    ("host1", "idc_a", 11.8, 10.3, 10.3, 1667446797450),
+    ("host1", "idc_a", 80.1, 70.3, 90.0, 1667446797550),
+    # ...... 22 rows
+    ("host1", "idc_b", 90.0, 39.9, 60.6, 1667446798250);
+```
+
+The size of the request is 950 bytes (38 x 25). The WCU of this request is 1. If you insert 40 rows in a single statement, the size is 1520 bytes (38 x 40), and the WCU of this request is 2.
+
+### RCU (Read Capacity Unit)
+
+Each API call to read data from your table is a read request. RCU is the server resource consumed in one request. It depends on the following items:
+
+- CPU time consumed by the request
```

Comment:
Please refer to https://github.com/GreptimeTeam/docs/pull/812/files
CPU time is no longer used.